liferay-echo/liferay-portal
# LPS-200973 Change property list keyboard navigation in segment editor

Motivation
=======
We want to unify the keyboard navigation with the one in page editor fragment list.
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-200973)


Steps to Verify:
----------------
1. Go to People -> Segments
1. Click on add new user segment
1. Navigate through the Properties sidebar using the keyboard and check that everything works as describe in the ticket.
